pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
-        tar/tar.gz: An archive file which contains the above directory format
-                    (must have .tar in the filename!)
+```
+$ sudo singularity shell --writable lolcow.img
+```
 
-    Targets can also be remote and defined by a URI of the following formats:
+### `--sandbox`
+If you wanted to create a container within a writable directory (called a _sandbox_) you could do so with the `--sandbox` option:
 
-        shub://     Build from a Singularity registry (Singularity Hub default)
-        docker://   This points to a Docker registry (Docker Hub default)
+```
+$ singularity build --sandbox lolcow/ shub://GodloveD/lolcow
 ```
 
-These "build spec options" refer to the [recipe bases](/docs-recipes#recipes) options that you have. 
-Generally you are going to be building into a container or folder from some base that is local (an image or directory) or
-from an external endpoint (e.g., docker). Next we can look at create options:
+The resulting directory operates just like a container in an image file.  You are permitted to make changes and write files within the directory, but those changes will not persist when you are finished using the container.  To make your changes persistent, use the `--writable` flag when you invoke your container.  
 
-## Development (writable) or Production (immutable)?
-When you build your container, you probably want a writable container for development, and a nice
-immutable one for your completed, production image. Here are the options under the usage (help) for build:
+Once again, it's a good idea to do this as root to ensure you have permission to access the files and directories that you want to change.
 
 ```
-CREATE OPTIONS:
-    -s|--sandbox    Build a sandbox rather then a read only compressed image
-    -w|--writable   Build a writable image (warning: deprecated due to sparse
-                    file image corruption issues)
-    -f|-F|--force   Force a rebootstrap of a base OS (note: this does not 
-                    delete what is currently in the image, just causes the core 
-                    to be reinstalled)
-    -T|--notest     Bootstrap without running tests in %test section
-    -s|--section    Only run a given section within the bootstrap (setup,
-                    post, files, environment, test, labels, none)
-
+$ sudo singularity shell --writable lolcow/
 ```
 
-You can think of the different options as follows:
+## Converting containers from one format to another
+If you already have a container saved locally, you can use it as a target to build a new container. This allows you convert containers from one format to another.  For example if you had a squashfs container called `production.simg` and wanted to convert it to a writeable ext3 container called `development.img` you could:
 
-**create writable development folder**
 ```
-sudo singularity build --writable container/ Singularity
+$ singularity build --writable development.img production.simg
 ```
```

Comment:
very good choice of container names here! :)

Reply:
thx